WORKSPACE

```diff
@@ -381,10 +381,12 @@ gapic_generator_ruby_repositories()
 # Discovery
 ##############################################################################
 
+_disco_to_proto3_converter_version = "91953d241824df417b6d4e7c2292e4622b4af432"
```

Comment:
Can we wait for this change to come from the sync?

Reply:
It does not matter which one comes first, but yes, there is a pending internal cl with this change (you are added as a reviewer)